apps/cli/src/node/docker-compose-validator.yaml

```diff
@@ -13,13 +13,21 @@ services:
             interval: 10s
             timeout: 1s
             retries: 5
-        user: root
+        post_start:
+            - command:
+                  - /bin/bash
+                  - -c
+                  - |
+                      mkdir -p "$CARTESI_SNAPSHOT_DIR"
+                      cp --recursive /tmp/snapshot/* "$CARTESI_SNAPSHOT_DIR"
+              user: root
         command:
             - /bin/bash
             - -c
             - |
-                mkdir -p "$CARTESI_SNAPSHOT_DIR"
-                cp --recursive /tmp/snapshot/* "$CARTESI_SNAPSHOT_DIR"
+                while ! stat "$CARTESI_SNAPSHOT_DIR" &>/dev/null; do
```

Comment:
This busy wait and coordination between the `command` and `post_start` is not very elegant.
Does it really need to be this way? What if the `post_start` fails for whatever reason?

Reply:
How about we first see how the `start` + `deploy` would work. Because it will be different in that case.

Reply:
sure, we can put this on-hold

if the rollups-node container image release already has this in-place, this process or preparing a directory should just be unnecessary